frontend/packages/console-shared/src/components/editor/YAMLEditorToolbar.tsx

```diff
@@ -1,17 +1,29 @@
 import * as React from 'react';
 import ShortcutsLink from './ShortcutsLink';
 import './YAMLEditorToolbar.scss';
+import { useTranslation } from 'react-i18next';
 
 interface YAMLEditorToolbarProps {
   showShortcuts?: boolean;
   toolbarLinks?: React.ReactNodeArray;
 }
 
 const YAMLEditorToolbar: React.FC<YAMLEditorToolbarProps> = ({ showShortcuts, toolbarLinks }) => {
+  const { t } = useTranslation();
   if (!showShortcuts && !toolbarLinks?.length) return null;
-
+  const isMac = window.navigator.platform.includes('Mac');
   return (
-    <div className="ocs-yaml-editor-toolbar__links">
+    <div className="ocs-yaml-editor-toolbar">
+      <span className="ocs-yaml-editor-shortcut__command">
+        <span className="ocs-shortcut__command">
+          <kbd>{isMac ? '⌥ Opt' : 'Alt'}</kbd>
+        </span>
```

Comment:
export `Command` as `ShortcutCommand` and reuse the existing component so that you aren't having to reference the css belonging to the other component.

Reply:
Updated to reuse component